src/dispatch/plugins/dispatch_slack/commands.py

```diff
@@ -161,50 +168,68 @@ def filter_tasks_by_assignee_and_creator(tasks: List[Task], by_assignee: str, by
     return filtered_tasks
 
 
-async def handle_slack_command(*, client, request, background_tasks):
-    """Handles slack command message."""
-    # We fetch conversation by channel id
-    channel_id = request.get("channel_id")
-
-    db_session = get_organization_from_channel_id(channel_id=channel_id)
-    # We get the name of command that was run
+async def handle_non_incident_conversation_commands(client, request, background_tasks):
+    """Handles all commands that do not have a specific incident conversation."""
     command = request.get("command")
+    channel_id = request.get("channel_id")
+    command_args = request.get("text", "").split(" ")
+    if len(command_args):
```

Comment:
```suggestion
    if len(command_args) > 0:
```

Reply:
This change would not be generally considered to be idiomatic python. See: https://medium.com/the-andela-way/idiomatic-python-coding-the-smart-way-cc560fa5f1d6

Reply:
According to this article, you should remove the `len` then.